### PR TITLE
Add some plugins

### PR DESCRIPTION
**Pull request by trynsta**
This PR adds:
- [Fire distortion](https://github.com/trynsta/awesome-linuxaudio/commit/4b32a5c783ea9c12a62c23baf063917a333cd530)
- [Cardinal](https://github.com/trynsta/awesome-linuxaudio/commit/f3290a3ed0e30f243411d6b2bd4c6a0cedb54c78)
- [LMMS assets](https://github.com/trynsta/awesome-linuxaudio/commit/7b7ff97df2b458db4c37e1a3a83fc87fedcd62a3)
- [RoomReverb](https://github.com/trynsta/awesome-linuxaudio/commit/c4cdc0053961f3e11af2718a79ebbfa811ac649d)
- [PeakEater](https://github.com/trynsta/awesome-linuxaudio/commit/3a7b0ef79e902c80481bbb51d2e53e2d55d59613)
- [MultiMeter](https://github.com/trynsta/awesome-linuxaudio/commit/f36c74947824a1d013bc95b9422728f309f7adbc)
- [valentine](https://github.com/trynsta/awesome-linuxaudio/commit/d91fcc3ba82b07157c416f32b982f0ccf7826df5)
- [airwin2rack](https://github.com/trynsta/awesome-linuxaudio/commit/0e300ce003a2ad700d70faf3f5ab8e60ee3d144e)
- [wolf-shaper](https://github.com/trynsta/awesome-linuxaudio/commit/36bfd051454c8504a5a15cadebdfd7b780f59222)
- [wolf-spectrum](https://github.com/trynsta/awesome-linuxaudio/commit/e068dc357ba2d07c9285a1e101a8ff371c822399)
- [centered by cozy-dsp](https://github.com/trynsta/awesome-linuxaudio/commit/cb2f56cc9e3e7fe7db4f54664cd46977d8b74f6d)
- [awesome-juce](https://github.com/trynsta/awesome-linuxaudio/commit/bb35f61be97b5fbfa50860e057fa68f89b71a360) audio plugins list
EDIT:
- [lamb-rc](https://github.com/nodiscc/awesome-linuxaudio/pull/67/commits/cc10c6460fa6e24ff3029f8fe3db261c09ca531e)

Also seems like current link for calf-plugins sourceforge is broken so I replaced that with link to their website.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -247,25 +250,31 @@ List keys:
 * [lv2vocoder](https://packages.debian.org/sid/lv2vocoder) - LV2 vocoder plugin ([◼](https://packages.debian.org/sid/lv2vocoder))
 * [mod-distortion](http://kxstudio.linuxaudio.org/Repositories:Plugins) - analog distortion emulation plugin `▒`
 * [mod-pitchshifter](http://kxstudio.linuxaudio.org/Repositories:Plugins) - Pitch Shifter LV2 effects `▒`
+* [MultiMeter](https://github.com/RealAlexZ/MultiMeter) - comprehensive set of AU/VST real-time audio analysis tools
```
Comment (nodiscc) [inline README.md]: Can you please move MultiMeter under the `Audio utilities - Meters & Analysis` section?

File: README.md [hunk 2]
```diff
@@ -247,25 +250,31 @@ List keys:
...
 * [terminatorx](http://www.terminatorx.org/) - realtime audio synthesizer ([◼](https://packages.debian.org/sid/terminatorx))
+* [valentine](https://github.com/tote-bag-labs/valentine) - open source compressor meant to pump and breathe
 * [vm.lv2](https://open-music-kontrollers.ch/lv2/vm/) - virtual machine plugin bundle (Control, CV, Audio, Atom) `▒`
 * [vocoder-ladspa](http://kxstudio.linuxaudio.org/Repositories:Plugins) - LADSPA Vocoder Plugin `▒`
 * [vocproc](http://hyperglitch.com/dev/VocProc) - LV2 plugin for pitch shifting and vocoding ([◼](https://packages.debian.org/sid/vocproc))
 * [wah-plugins](http://kokkinizita.linuxaudio.org/linuxaudio/index.html) - auto-wah LADSPA plugin ([◼](https://packages.debian.org/sid/wah-plugins))
 * [wasp-ladspa](http://kxstudio.linuxaudio.org/Repositories:Plugins) - WASP LADSPA plugin `▒`
+* [wolf-shaper](https://wolf-plugins.github.io/wolf-shaper/) - Waveshaper plugin with a spline-based graph editor
+* [wolf-spectrum](https://github.com/wolf-plugins/wolf-spectrum) - Real-time 2D spectrogram plugin
```
Comment (nodiscc) [inline README.md]: Please also move wolf-spectrum under `Audio utilities - Meters & Analysis`

File: README.md [hunk 3]
```diff
@@ -818,6 +828,7 @@ _Sample segmentation, analysis, programmable editors_
 * [Freesound](http://www.freesound.org/browse/) - a huge collaborative database of audio snippets, samples, recordings, bleeps, ... released under Creative Commons licenses that allow their reuse.
 * [CCMixter](http://ccmixter.org/) - Search engine for Creative Commons multimedia content
 * [Freepats project](http://freepats.zenvoid.org/) - Free and open instrument sets, samples, sound synthesis, music...
+* [assets](https://github.com/LMMS/assets) - Samples, presets, projects and patches for LMMS
```
Comment (nodiscc) [inline README.md]:
```suggestion
* [LMMS assets](https://github.com/LMMS/assets) - Samples, presets, projects and patches for LMMS
```

**Review by nodiscc — CHANGES_REQUESTED** [review]
Some very nice additions in there, thank you!

A few suggested changes below

File: README.md (hunk 2 above)
Comment (trynsta) [reply README.md]: You are welcome. I've done what you asked for.

**Review by nodiscc — APPROVED** [review]
Thank you

**Review by saydselim01 — APPROVED** [review]

**Review by saydselim01 — APPROVED** [review]

**Review by saydselim01 — APPROVED** [review]

**Review by saydselim01 — APPROVED** [review]

**Pull request merged**